src/fontra/views/editor/scene-controller.js

```diff
@@ -1168,6 +1181,28 @@ export class SceneController {
     });
   }
 
+  async doJoinSelectedOpenContours() {
+    let newSelection = new Set();
+    const [pointIndex1, pointIndex2] = this.contextMenuState.joinContourSelection;
+    await this.editLayersAndRecordChanges((layerGlyphs) => {
+      for (const layerGlyph of Object.values(layerGlyphs)) {
+        const selectionPointIndices = connectTwoDistinctContours(
+          layerGlyph.path,
+          pointIndex1,
+          pointIndex2
+        );
+
+        for (const pointIndex of selectionPointIndices) {
+          newSelection.add(`point/${pointIndex}`);
+        }
+      }
+      if (newSelection) {
```

Comment:
Since you initialize `newSelection` with an empty set, this will always be true. Do you intend to do this only on a non-empty selection, or is it a safe-guard against undefined/null?

Reply:
I removed the unnecessary check and also made it `const`. done.